study.md

```diff
@@ -38,29 +38,71 @@ As it makes sense to you please write the following in JSON format:
 -   Has lived in two places recently: 123 Fake street, and 2 Muchinfo Road
 
 ```json
-<!-- your answer here -->
+{
+  "jason" : {
+    "cat" : "Mr. Kitty",
+    "Roomates": {
+      "first": "Dave",
+      "second": "Miller"
+    },
+    "love":{
+      "loveOne": "Red Sox",
+      "loveTwo": "Patriots",
+      "loveThree":"Bruins"
+    },
+    "places": {
+      "firstPlace": "123 Fake street",
+      "secondPlace": "2 Muchinfo Road"
+    }
+
+  }
+
+}
 ```
 
 I forgot, my roomate Dave has two goldfish, one named Bob (he's red) and the
 other named Mr.MagicNibbles (yellow) and bites a bit. Please copy the JSON you
 already wrote and include the information about Dave's fish.
 
 ```json
-<!-- your answer here -->
+{
+  "jason" : {
+    "cat" : "Mr. Kitty",
+    "Roomates": {
+      "first": "Dave",
+      "second": "Miller"
+    },
+    "Dave's fishes":{
+      "redFishName": "Bob",
+      "yellowFishName":"Dave"
+    },
+    "love":{
+      "loveOne": "Red Sox",
+      "loveTwo": "Patriots",
+      "loveThree":"Bruins"
```

Comment:
do you think this naming convention for love makes sense?

What if Jason loves a million things?